### ISSUE TITLE
Why do I try many times, but the results of images are always blurry?

### ISSUE THREAD

**Issue by wzhdsg** [thfylsty/ICPR2020_A_Dual_branch_Network_for_Infrared_and_Visible_Image_Fusion#1]
I trained 500 epoches.    No code modification at all.

**Comment by thfylsty** [comment]
I'm not quite sure what the issue you mentioned is caused by. It could be an issue with the training set, or it could be that it hasn't converged yet. I suggest printing and observing the change in loss value. If it's caused by a convergence issue, I recommend lowering the learning rate.

**Comment by thfylsty** [comment]
If that still doesn't work, I recommend disabling some of the loss functions and testing the training performance with the basic loss function first.

**Comment by bingkun99** [comment]
> I trained 500 epoches. No code modification at all.
Are you using the TNO data set?
